_includes/head.html

```diff
@@ -1,12 +1,11 @@
 <head>
   <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-27002928-1"></script>
+  <script async src="https://www.googletagmanager.com/gtm.js?id=GTM-K7GFJTV"></script>
```

Comment:
Updating this to use the right tracking using Google Tag Manager

Slack convo: https://zapier.slack.com/archives/C028W5V2A/p1559853829094900?thread_ts=1559851688.091600&cid=C028W5V2A